main/integration-tests/README.md

```diff
@@ -45,22 +48,32 @@ Once test resources and config file are created, you can run the integration tes
 Note: Integration tests will create resources in the environment they are executed against.
 
 ### Run against dev environment
+
 - In config file `main/integration-tests/config/settings/<STAGE>.yml`
   - set `isBuildServer` to `false`
   - set `isLocal` to `false`
  
 ##### run all integration tests from the root directory with: 
+
+Run AppStream tests. AppStream and Egress should be enabled in the testing environment
+```bash
+$ scripts/run-integration-tests.sh <STAGE> AppStream
```

Comment:
Should we include egress in the stage- `AppStreamEgressEnabled`?

Reply:
updated